web-client/src/pages/CheckoutPage.tsx

```diff
@@ -0,0 +1,210 @@
+                                control={form.control}
+                                name="state"
+                                render={({ field }) => (
+                                    <FormItem className="w-1/2">
+                                        <FormLabel className="font-bold">State</FormLabel>
+                                        <FormControl>
+                                            <DropdownMenu>
+                                                <DropdownMenuTrigger asChild>
+                                                    <Button variant="outline" className="w-full">
+                                                        {field.value || "Select a state"}
+                                                    </Button>
+                                                </DropdownMenuTrigger>
+                                                <DropdownMenuContent className="w-full max-h-60 overflow-auto">
+                                                    {states.map((state) => (
+                                                        <DropdownMenuItem
+                                                            key={state}
+                                                            onSelect={() => field.onChange(state)}
+                                                        >
+                                                            {state}
+                                                        </DropdownMenuItem>
+                                                    ))}
+                                                </DropdownMenuContent>
+                                            </DropdownMenu>
+                                        </FormControl>
+                                        <FormMessage />
+                                    </FormItem>
+                                )}
+                            />
+
+                            <FormField
+                                control={form.control}
+                                name="zipcode"
+                                render={({ field }) => (
+                                    <FormItem className="w-1/2">
+                                        <FormLabel className="font-bold">Zipcode</FormLabel>
+                                        <FormControl>
+                                            <Input {...field} className="w-full" />
+                                        </FormControl>
+                                        <FormMessage />
+                                    </FormItem>
+                                )}
+                            />
+                        </div>
+
+                        <p className="text-gray-500 mt-8 mb-8">
+                            Shipping information and order updates will be sent to your email address.
+                        </p>
+
+                        <Button type="submit" className="w-full bg-black text-white">Pay with Stripe</Button>
```

Comment:
You can remove the custom styles and just leave the default button :)

Reply:
Fixed!